src/components/button/Button.tsx

```diff
@@ -58,18 +68,33 @@ export const Button = ({
   const mode = primary
     ? 'storybook-button--primary'
     : 'storybook-button--secondary';
+  if (backgroundColor) {
+    document.documentElement.style.setProperty('--custom', backgroundColor);
```

Comment:
dont use vanilla js here use virtual dom instead of dom directly

Reply:
Comment dismissed: Other comments changes what this refers to.